app/javascript/react/pages/CalendarPage/CalendarPage.tsx

```diff
@@ -72,7 +72,11 @@ const CalendarPage: React.FC<CalendarPageProps> = ({ children }) => {
       .subtract(2, "months")
       .format("YYYY-MM-DD");
 
-    console.log("Downloading first time - moving stream data. Start - End: ", newStartDate, formattedEndDate)
+    console.log(
+      "Downloading first time - moving stream data. Start - End: ",
+      newStartDate,
+      formattedEndDate
+    );
```

Comment:
Is it for debugging or do we want it to stay?

Reply:
"It's Janek's code :)"